docs/components/ListView.jsx

```diff
@@ -0,0 +1,231 @@
+            ]}
+            value={itemsKey}
+          />
+        </div>
+        <label className={cssClass.CONFIG}>
+          <input
+            type="checkbox"
+            checked={showBorder}
+            onChange={e => this.setState({showBorder: e.target.checked})}
+          />
+          {" "}
+          Show Border
+        </label>
+        <label className={cssClass.CONFIG}>
+          <input
+            type="checkbox"
+            checked={showTitle}
+            onChange={e => this.setState({showTitle: e.target.checked})}
+          />
+          {" "}
+          Show Title
+        </label>
+        <label className={cssClass.CONFIG}>
+          <input
+            type="checkbox"
+            checked={showCustomEmptyMessage}
+            onChange={e => this.setState({showCustomEmptyMessage: e.target.checked})}
+          />
+          {" "}
+          Custom Empty Message
+        </label>
+      </FlexBox>
+    );
+  }
+
+  render() {
+    const {Item} = List;
+    const {cssClass} = ListView;
+    const {itemsKey, rowType, showBorder, showCustomEmptyMessage, showTitle} = this.state;
+    const items = Items[itemsKey];
+
+    return (
+      <View
+        className={cssClass.CONTAINER}
+        title="List"
+        sourcePath="src/List/List.jsx"
+      >
+        <div className={cssClass.INTRO}>
+          <p>
+            Provides a simple view for a list of items with various display configuration options.
+          </p>
+          <p>
+            Closely follows <Link to="/components/table">Table</Link> component styling.
+          </p>
+          <CodeSample>
+            {`
+              import {List} from "clever-components";
+              const {Item, RowType} = List;
+            `}
+          </CodeSample>
+        </div>
+
+        <Example title="Basic Usage:">
+          <List className={cssClass.LIST} title={"Days of the Week (That Matter)"}>
+            <Item>Friday</Item>
+            <Item>Saturday</Item>
+            <Item>Sunday</Item>
+          </List>
+        </Example>
+
+        <Example title="With Customization Options:">
+          <ExampleCode>
+            <List
+              className={cssClass.LIST}
+              emptyMessage={showCustomEmptyMessage ? "I hate everything." : undefined}
+              noBorder={!showBorder}
+              rowType={rowType}
+              title={showTitle ? "A Few of My Favourite Things" : undefined}
+            >
+              {items.map((item, i) => (
+                <Item className="my--custom--class" key={i}>
+                  {item}
+                </Item>
+              ))}
+            </List>
+          </ExampleCode>
+          {this.renderConfig()}
+        </Example>
+
+        <PropDocumentation
+          title="<List /> Props"
+          availableProps={[
+            {
+              name: "children",
+              type: "List.Item | List.Item[]",
+              description: "List items to display.",
+              optional: true,
```

Comment:
yeah, fair point. functionally, it's optional, but in practice, there's no reason for anyone to provide null/undefined children